website/src/pages/StatusPage/components/ExecutionQueue/ExecutionQueue.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { ExecutionQueueType, columns } from "./Columns";
+import { data } from "./data.json";
```

Comment:
Let's not forget to actually do the API call before merging this. I will review one last time with the API call before merging.